modules/OCPPConfiguration/config-mapping.yaml

```diff
@@ -0,0 +1,312 @@
+- component:
```

Comment:
This file should serve as an example. 
* Reduce entries in this file and rename it to `example_mapping.yaml`
* add reasonable example for OCPP1.6 (e.g. ConnectionTimeOut maps to module_id: auth, config_param: connection_timeout)
* add reasonable example for OCPP2.0.1 (e.g. Component=TxCtrlr, Variable=EVConnectionTimeOut maps to module_id: auth, config_param: connection_timeout

Reply:
Done